### PR TITLE
Update Trust Registry Service docs

### PR DESCRIPTION
**Pull request by chiarapaskulin**

### CODE REVIEW

**Review by lucasamonrc — APPROVED** [review]

File: docs/reference/services/trust-registry-service.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Trust Registry Service
 
-The Trust Registry Service exposes functionality for managing [Trust Registries](/learn/concepts/trust-registries) -- lists of authorized issuers for the various credential types within an ecosystem.
+The Governance API exposes functionalities for managing [Trust Registries](/learn/concepts/trust-registries).
```
Comment (geel9) [inline docs/reference/services/trust-registry-service.md]:
```suggestion
The Governance API exposes functionality for managing your Ecosystem's [Trust Registry](/learn/concepts/trust-registries).
```

File: docs/reference/services/trust-registry-service.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+
+A Trust Registry is a list of authorized issuers for the various credential types within an ecosystem. Each ecosystem has a Trust Registry and you can use the following API calls to manage its trusted members - register, unregister, list, check status.
```
Comment (geel9) [inline docs/reference/services/trust-registry-service.md]:
```suggestion
A Trust Registry is a list of authorized issuers for the various credential types within an ecosystem.
```

I think less is more, here

**Review by geel9 — CHANGES_REQUESTED** [review]

File: docs/reference/services/credential-service.md
Comment (chiarapaskulin) [inline docs/reference/services/credential-service.md:1]: The redirect was breaking in the docs with the ".md"

File: docs/reference/services/credential-service.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ Here's an example of a VC with extended issuer information:
   }
 ```
 
-See [Trust Registry Service](/reference/services/trust-registry-service.md) for more information on using governance in your ecosystem.
+See [Trust Registry Service](/reference/services/trust-registry-service) for more information on managing your ecosystem's governance.
```
Comment (chiarapaskulin) [inline docs/reference/services/credential-service.md:84]: Redirect in the website was failing with the ".md" included.

File: docs/reference/services/credential-service.md [hunk 4]
```diff
@@ -65,10 +65,10 @@ Issues a credential from a [previously-defined template](/reference/services/tem
...
 
 In order to attach governance information to the credential, issuers must request this explicitly by specifying the parameter
-`framework_id` in the above request. The framework identifier references the ecosystem governance framework that the issuer is authorized
-to issue credentials of the designated type (schema). This identifier has the format `urn:egf:<ecosystem>:<type>`.
+`include_governance` to `true` in the above request. This will reference the ecosystem's governance framework that the issuer is authorized
+to issue credentials of the designated type (schema).
 
-When this parameter is specified, the issued credential will contain extended information in the `issuer` field to assert authorization in
+When this parameter is set to `true`, the issued credential will contain extended information in the `issuer` field to assert authorization in
 the given governance framework. The framework identifier will be in the `issuer.trustRegistry` field.
```
Comment (chiarapaskulin) [inline docs/reference/services/credential-service.md:72]: @geel9 Is the framework identifier still stored in the credential after the related API changes?

Comment (geel9) [reply docs/reference/services/credential-service.md:72]: Yes, it is

**Review by geel9 — APPROVED** [review]

**Review by fundthmcalculus — APPROVED** [review]

**Pull request merged**